utils/task/converter.ts

```diff
@@ -0,0 +1,13 @@
+import { Task } from '@prisma/client';
+import { TaskData } from '../../types/task';
+
+export function convertTaskToPayload(task: Task): TaskData {
+  const { id, title, dueDate, notificationDateTime, isDone } = task;
+  return {
+    id,
+    title,
+    dueDate: dueDate.toJSON() ?? null,
```

Comment:
I don't think dueDate is supposed to be null?
```suggestion
    dueDate: dueDate.toJSON(),
```

Reply:
Oh, thanks for the catch!